include/ebpf_program_types.h

```diff
@@ -37,7 +37,7 @@ typedef struct _ebpf_helper_function_prototype
     ebpf_argument_type_t arguments[5];
 } ebpf_helper_function_prototype_t;
 
-typedef struct _ebpf_program_info
+typedef struct bpf_prog_info
```

Comment:
Why bpf_prog_info and not _bpf_prog_info? I thought style guidelines had the struct name start with _?

Reply:
Per our discussion this week, style guidelines are:
1) If the type is public in libbpf, match it literally,
2) else struct names start with _

In this case, rule 1 applies.

Reply:
Even though there is a "bpf_prog_info" in libbpf.h, it is not same as the one defined here. We had added ebpf_program_info structure to return program information from EC using the APIs `ebpf_get_program_info()` and `ebpf_get_program_info_by_fd`

Reply:
thanks, reverted